backend/qemu.pm

```diff
@@ -978,7 +978,9 @@ sub start_qemu {
         }
         if ($vars->{QEMU_NUMA}) {
             for my $i (0 .. ($vars->{QEMUCPUS} - 1)) {
-                sp('numa', [qv "node nodeid=$i"]);
+                my $m = $vars->{QEMURAM} / $vars->{QEMUCPUS};
```

Comment:
This won't work if `QEMURAM` is not divisible by `QEMUCPUS`. You should round the value down and add the remainder to one or more nodes.